ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/NewWallet.hs

```diff
@@ -135,26 +103,16 @@ initNewWallet = do
   (walletMnemonicWidget, walletMnemonicLayout) <- createSubWidget
   addRow walletMnemonicLayout walletMnemonicLabel walletMnemonic
 
-  fullRestoreLabel <- QLabel.newWithText ("Perform full restore" :: String)
-  fullRestore <- QCheckBox.new
+  fullRestore <- createCheckBox walletMnemonicLayout CheckboxOnRight "Perform full restore"
   QAbstractButton.setChecked fullRestore True
-  QWidget.setSizePolicyRaw fullRestore Maximum Preferred
-
-  addRow walletMnemonicLayout fullRestoreLabel fullRestore
 
   QBoxLayout.addWidget layout walletMnemonicWidget
   QWidget.hide walletMnemonicWidget
 
   addSeparator layout
 
-  hasPasswordLabel <- QLabel.newWithText $ toString passwordLabelText
-  QLabel.setWordWrap hasPasswordLabel True
-  QWidget.setMinimumSizeRaw hasPasswordLabel 600 30
-  hasPassword <- QCheckBox.new
+  hasPassword <- createCheckBox layout CheckboxOnRight passwordLabelText
```

Comment:
Everything related to passwords in this dialog is now useless and should probably be removed. This dialog creates new wallets with `UiRestoreWallet` command and you now ignore its password field.